acceptance/repository-cloudant/README.md

```diff
@@ -45,27 +39,25 @@ same (except couchDB doesn't support geosearch).
 
 - Install [Docker](https://docs.docker.com/engine/installation/).
 
-- Use the `couchDB3:latest` image, and rename the container as
-  `cloudant-testdb`.
-
-- Start the container, and create an admin and password for this image at
-  `http://localhost:{YOURPORT}/_utils/` if it doesn't have any yet.
-
-- Then run the following script
+- Run the following script
 
 ```bash
-source setup.sh <HOST> <PORT> <USER> <PASSWORD> <DATABASE>
+CLOUDANT_USER=<USER> CLOUDANT_PASSWORD=<PASSWORD> CLOUDANT_DATABASE=<DATABASE> node docker.setup.sh
```

Comment:
`node docker.setup.sh` looks suspicious - node typically expects `.js` files, not `.sh`. I think it's just a typo?

```suggestion
CLOUDANT_USER=<USER> CLOUDANT_PASSWORD=<PASSWORD> CLOUDANT_DATABASE=<DATABASE> node docker.setup.js
```

BTW why `node docker.setup.js` and not `npm run docker:setup`? Should we modify Travis CI config to call `node docker.setup.js` too?

Reply:
Good catch!

And right, I think 
```
CLOUDANT_USERNAME=<USER> CLOUDANT_PASSWORD=<PASSWORD> CLOUDANT_DATABASE=<DATABASE> npm run docker:setup
```
 also allows users to have their own configurations 👍 changed.